spec/rails_helper.rb

```diff
@@ -6,6 +6,7 @@
 require "spec_helper"
 require "rspec/rails"
 require "capybara/rails"
+require 'database_cleaner'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.